src/Reshuffle.ts

```diff
@@ -113,6 +115,9 @@ export default class Reshuffle {
           const httpMultiplexer = this.httpDelegates[path]
           webserver.all(path, httpMultiplexer.handle.bind(httpMultiplexer))
         })
+      webserver.use(HEALTH_CHECK_PATH, (req, res) =>
```

Comment:
is there a way to tell Reshuffle that you do not want a health check?

Reply:
done, only make health check available when HEALTH_CHECK_PATH is set.
It's covered by the unit tests